fdbserver/ClusterController.actor.cpp

```diff
@@ -991,8 +1011,49 @@ class ClusterControllerData {
 		return false;
 	}
 
+	void updateUsedIds(RegisterMasterRequest const& req) {
```

Comment:
change this to generate an idUsed from serverDBInfo, and only call it before using idUsed (so do not store idUsed in ClusterControllerData)